packages/grid/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -495,9 +495,13 @@ export interface DataGridPropsWithoutDefaultValue<R extends GridValidRowModel =
    * Callback fired when a cell loses focus.
    * @param {GridCellParams} params With all properties from [[GridCellParams]].
    * @param {MuiEvent<MuiBaseEvent>} event The event object.
-   * @param {GridCallbackDetails} details Additional details for this callback.
    */
   onCellFocusOut?: GridEventListener<'cellFocusOut'>;
+  /**
+   * Callback fired when a cell gains focus.
+   * @param {GridCellParams} params With all properties from [[GridCellParams]].
```

Comment:
We are missing the event argument

Reply:
It looks like we only pass `params` to this event:

https://github.com/mui/mui-x/blob/ffeb700ae08cee2e69e977066881e42a64ec7013/packages/grid/x-data-grid/src/hooks/features/focus/useGridFocus.ts#L82

https://github.com/mui/mui-x/blob/ffeb700ae08cee2e69e977066881e42a64ec7013/packages/grid/x-data-grid/src/models/events/gridEventLookup.ts#L443

Reply:
Same as my other comment, `publishEvent` always adds an event.
If none is passed to him, it will be an object with just the `defaultMuiPrevented` property.

Reply:
Oh, it's also added by `publishEvent`
https://github.com/mui/mui-x/blob/ffeb700ae08cee2e69e977066881e42a64ec7013/packages/grid/x-data-grid/src/hooks/core/useGridApiInitialization.ts#L33-L42

Reply:
@flaviendelangle thanks for pointing it out!